commandline/src/main/java/tech/pegasys/web3signer/commandline/Web3SignerBaseCommand.java

```diff
@@ -91,7 +91,7 @@
       paramLabel = "<LOG VERBOSITY LEVEL>",
       description =
           "Logging verbosity levels: OFF, FATAL, WARN, INFO, DEBUG, TRACE, ALL (default: INFO)")
-  private final Level logLevel = Level.INFO;
+  private final Level logLevel = null;
```

Comment:
are we not defaulting to INFO anymore?

Reply:
The defaulting is managed in the log4j2.xml file in under app